config/locales/en.yml

```diff
@@ -847,6 +847,7 @@ en:
         is_required: "Required"
         max_length: "Maximum length"
         min_length: "Minimum length"
+        content_right_to_left: "Left-to-Right content"
```

Comment:
Can you double check that the translation is correct? The key is "right-to-left", but the translation is "left-to-right".